packages/gettext_tiny.rb

```diff
@@ -8,11 +8,15 @@ class Gettext_tiny < Package
   source_url 'http://ftp.barfooze.de/pub/sabotage/tarballs/gettext-tiny-0.3.2.tar.xz'
   source_sha256 'a9a72cfa21853f7d249592a3c6f6d36f5117028e24573d092f9184ab72bbe187'
 
+  def self.preinstall
+    abort "gettext already installed. gettext_tiny conflics with gettext.".lightred if File.exist? "#{CREW_PREFIX}/bin/msgcomm"
```

Comment:
conflicts?

Reply:
they can't be installed at the same time. they both provide binaries/libraries in the same place and would overwrite each other

Reply:
No, I meant you misspelled conflicts. :)

Reply:
oh, thank you